pkg/core/native/notary.go

```diff
@@ -272,7 +272,7 @@ func (n *Notary) lockDepositUntil(ic *interop.Context, args []stackitem.Item) st
 		return stackitem.NewBool(false)
 	}
 	till := toUint32(args[1])
-	if till < ic.BlockHeight() {
+	if till < ic.BlockHeight() { // @roman-khimov, is it OK to lock deposit till the current persisting block?
```

Comment:
We need to decide about this one. It's not critical to have it as it is now.

Reply:
Yes, the other way to look at it is
https://github.com/nspcc-dev/neo-go/blob/f10d2d2be7de5927f532a177cc09f63877b121e5/pkg/core/native/notary.go#L311-L313
If you deposit, but can withdraw immediately the deposit is useless.

Reply:
Good, I've fixed this in couple of places and added a comment to `withdraw`. Could you check it?